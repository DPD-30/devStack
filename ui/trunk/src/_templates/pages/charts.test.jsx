import React from 'react'
import { renderToString } from 'react-dom/server'

import PagesAdminCharts from './charts.jsx'

jest.unmock('./charts.jsx')


describe('', () => {
    it('PagesAdminCharts should match snapshot', () => {
        const comp = renderToString(<PagesAdminCharts />)

        expect(comp).toMatchSnapshot()
    })
})
