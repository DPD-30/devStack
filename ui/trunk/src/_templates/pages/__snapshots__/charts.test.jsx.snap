// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PagesAdminCharts should match snapshot 1`] = `
<div data-reactroot>
</div>
`;
